launch_testing/test/launch_testing/examples/hello_world_launch_test.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import launch
+import launch.actions
+import launch_testing.actions
+import launch_testing.markers
+import pytest
+
+
+@pytest.mark.launch_test
+@launch_testing.markers.keep_alive
+def generate_test_description():
+    """Launch a simple process to print 'hello_world'."""
+    return launch.LaunchDescription([
+        launch.actions.ExecuteProcess(
+            cmd=['echo', 'hello_world']
+        ),
+        launch_testing.actions.ReadyToTest()
```

Comment:
If we consider this may be the first example a user sees, I think we should annotate the code more with comments, e.g.
```suggestion
        # Launch a process to test
        launch.actions.ExecuteProcess(
            cmd=['echo', 'hello_world']
        ),
        # Tell launch to start tests
        launch_testing.actions.ReadyToTest()
```